package.json

```diff
@@ -52,7 +52,9 @@
 		"docker:clean": "yarn docker:compose down --rmi all -v && rm -rf docker/wordpress/* docker/wordpress/.htaccess docker/wordpress-develop/* docker/logs/* docker/data/mysql/*",
 		"docker:env": "node -e \"require('fs').createWriteStream( 'docker/.env', { flags: 'a' } );\"",
 		"docker:update-core-unit-tests": "yarn docker:compose exec wordpress svn up /tmp/wordpress-develop/tests/phpunit/data/ /tmp/wordpress-develop/tests/phpunit/includes",
-		"lint": "eslint --ext .js,.jsx . _inc extensions modules",
+		"lint": "npm-run-all --parallel lint:*",
+		"lint:js": "eslint -c .eslintrc.js --ext .js,.jsx . _inc extensions modules",
+		"lint:css": "stylelint \"**/*.scss\" --syntax scss",
```

Comment:
Maybe we should call it `lint:scss` instead, since we only target scss files for now?

Reply:
Valid! :-) https://github.com/Automattic/jetpack/pull/11767/commits/46ff18c80c5277b20e6f43144fc36cfcdfea7a50